base/uk.ac.stfc.isis.ibex.opis/resources/DG645/dg645.opi

```diff
@@ -1966,6 +1945,495 @@ $(pv_value)</tooltip>
+          <x>0</x>
+          <y>38</y>
+        </widget>
+        <widget typeId="org.csstudio.opibuilder.widgets.ActionButton" version="2.0.0">
+          <actions hook="false" hook_all="false">
+            <action type="WRITE_PV">
+              <pv_name>$(pv_name)</pv_name>
+              <value>1</value>
+              <timeout>10</timeout>
+              <confirm_message>Are you sure you want to save? This will overwrite any data in the selected slot. Note: slot 0 cannot be written to.</confirm_message>
+              <description></description>
+            </action>
+          </actions>
+          <alarm_pulsing>false</alarm_pulsing>
+          <backcolor_alarm_sensitive>false</backcolor_alarm_sensitive>
+          <background_color>
+            <color red="240" green="240" blue="240" />
+          </background_color>
+          <border_alarm_sensitive>false</border_alarm_sensitive>
+          <border_color>
+            <color red="0" green="128" blue="255" />
+          </border_color>
+          <border_style>0</border_style>
+          <border_width>1</border_width>
+          <enabled>true</enabled>
+          <font>
+            <opifont.name fontName="Segoe UI" height="9" style="0" pixels="false">Default</opifont.name>
+          </font>
+          <forecolor_alarm_sensitive>false</forecolor_alarm_sensitive>
+          <foreground_color>
+            <color red="0" green="0" blue="0" />
+          </foreground_color>
+          <height>25</height>
+          <image></image>
+          <name>Save_Button</name>
+          <push_action_index>0</push_action_index>
+          <pv_name>$(PV_ROOT)SAVE</pv_name>
+          <pv_value />
+          <rules />
+          <scale_options>
+            <width_scalable>true</width_scalable>
+            <height_scalable>true</height_scalable>
+            <keep_wh_ratio>false</keep_wh_ratio>
+          </scale_options>
+          <scripts />
+          <style>0</style>
+          <text>Save</text>
+          <toggle_button>false</toggle_button>
+          <tooltip>$(pv_name)
+$(pv_value)</tooltip>
+          <visible>true</visible>
+          <widget_type>Action Button</widget_type>
+          <width>61</width>
+          <wuid>-4d9fb07c:18a031ecdc2:-7dcc</wuid>
+          <x>162</x>
+          <y>38</y>
+        </widget>
+        <widget typeId="org.csstudio.opibuilder.widgets.ActionButton" version="2.0.0">
+          <actions hook="false" hook_all="false">
+            <action type="WRITE_PV">
+              <pv_name>$(pv_name)</pv_name>
+              <value>1</value>
+              <timeout>10</timeout>
+              <confirm_message>Are you sure you want to load configuration from the selected slot? This will overwrite current settings, and restart the IOC. NOTE: Changes may take a few minutes to update, do not touch any other buttons until after the IOC restarts. The Value 1 in this dialogue box does not reflect the selected slot.</confirm_message>
```

Comment:
You could change this to include message about value displayed in the dialog too